utils/scripts/ocm/ocm.py

```diff
@@ -172,28 +182,46 @@ def osd_cluster_create(self):
                     "Invalid channel group. Values can be 'stable' or 'candidate'."
                 )
 
-        cmd = (
-            "ocm create cluster --aws-account-id {} "
-            "--aws-access-key-id {} --aws-secret-access-key {} "
-            "--ccs --region {} --compute-nodes {} "
-            "--compute-machine-type {} {} {}"
-            "{}".format(
-                self.aws_account_id,
-                self.aws_access_key_id,
-                self.aws_secret_access_key,
-                self.aws_region,
-                self.num_compute_nodes,
-                self.aws_instance_type,
-                version,
-                channel_grp,
-                self.cluster_name,
+        if self.cloud_provider == "aws":
+            cmd = (
+                "ocm create cluster --aws-account-id {} "
+                "--aws-access-key-id {} --aws-secret-access-key {} "
+                "--ccs --region {} --compute-nodes {} "
+                "--compute-machine-type {} {} {}"
+                "{}".format(
+                    self.aws_account_id,
+                    self.aws_access_key_id,
+                    self.aws_secret_access_key,
+                    self.aws_region,
+                    self.num_compute_nodes,
+                    self.aws_instance_type,
+                    version,
+                    channel_grp,
+                    self.cluster_name,
+                )
+            )
+        elif self.cloud_provider == "gcp":
+            # Create service account file
+            self._create_service_account_file()
+            cmd = (
+                "ocm create cluster --provider {} --service-account-file {} "
+                "--ccs --region {} --compute-nodes {} "
+                "--compute-machine-type {} {} {}"
+                "{}".format(
+                    self.cloud_provider,
+                    self.service_account_file,
+                    self.region,
+                    self.compute_nodes,
```

Comment:
is it `compute_nodes` the number of nodes? if yes, could we use `num_compute_nodes` like done with AWS? so we don't have 2 variables for the same value

Reply:
It is same berto. But as per argparse logic written below, it needs to be 2 different variable. I will take this as enhancement and send a new PR for this.